ansible/roles/commcarehq/tasks/celery.yml

```diff
@@ -0,0 +1,73 @@
+- name: define reminder queue services
+  become: cchq
+  template:
+    src: "../templates/common_supervisor_management.conf.j2"
+    dest: "{{ service_home }}/{{ deploy_env }}_{{ item.dest_name }}"
+  when: app_processes_config.celery_processes.get(inventory_hostname).reminder_queue is defined
+  with_items:
+    - dest_name: supervisor_reminder_queue.conf
+      process_name: reminder_queue
+      management_command: run_reminder_queue
+    - dest_name: supervisor_queue_schedule_instances.conf
+      process_name: queue_schedule_instances
+      management_command: queue_schedule_instances
+    - dest_name: supervisor_handle_survey_actions.conf
+      process_name: handle_survey_actions
+      management_command: handle_survey_actions
+  tags:
+    - services
+  register: celery_reminder_queue_files
```

Comment:
It's may be my incorrect understanding, but isn't `reminder_queue`, `sms_queue` etc that are not set via `set_celery_supervisorconf` in `set_supervisor_config` not related to celery and just management commands that need to be run periodically? Just confused by the naming here with `celery_`

Reply:
Yeha they aren't technically celery queues, which I didn't realize until I was writing htis PR. Historically they've been specified under "celery" in the app-processes.yml file though, so I kept that convention here